Pear/Profile/ProfilePillCollectionViewCell.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ProfilePillCollectionViewCell.swift
+//  Pear
+//
+//  Created by Lucy Xu on 3/13/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class ProfilePillCollectionViewCell: UICollectionViewCell {
+
+    // MARK: - Private View Vars
+    private let label = UILabel()
+    private let imageView = UIImageView()
+
+    static let reuseIdentifier = "ProfilePillCollectionViewCell"
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        clipsToBounds = false
+        layer.cornerRadius = 18
+        backgroundColor = .paleGreen
+
+        label.textAlignment = .left
+        label.font = ._16CircularStdBook
+        label.preferredMaxLayoutWidth = 120
+        label.numberOfLines = 0
+        label.textColor = .black
+        contentView.addSubview(label)
+
+        contentView.addSubview(imageView)
+
+        imageView.snp.makeConstraints { make in
+            make.size.equalTo(CGSize(width: 20, height: 20))
+            make.top.bottom.equalToSuperview().inset(6)
+            make.leading.equalToSuperview().inset(12)
+        }
+
+        label.snp.makeConstraints { make in
+            make.leading.equalTo(imageView.snp.trailing).offset(6)
+            make.top.bottom.trailing.equalToSuperview()
```

Comment:
I don't know what the design should be, but just double check that the trialing inset shouldn't be 6 as well.